src/Navbar.jsx

```diff
@@ -47,23 +47,32 @@ const Navbar = ({ token, deleteToken }) => {
         <StyledListItem>
           <StyledLink to="/add">Add Conference</StyledLink>
         </StyledListItem>
-        <StyledListItem>
-          <StyledLink to="/users/register">Register</StyledLink>
-        </StyledListItem>
-        <StyledListItem>
-          {token ? (
-            <Button
-              navLink
-              onClick={() => {
-                logout();
-              }}
-            >
-              Logout
-            </Button>
-          ) : (
-            <StyledLink to="/users/login">Login</StyledLink>
-          )}
-        </StyledListItem>
+        {token ? (
+          <>
+            <StyledListItem>
+              <StyledLink to={`/users/${token.userId}`}>Profile</StyledLink>
+            </StyledListItem>
+            <StyledListItem>
+              <Button
+                navLink
+                onClick={() => {
```

Comment:
You should just be able to pass logout in here, instead of returning it in the arrow function

Reply:
This causes the logout function to trigger on every page refresh

Reply:
You should be able to just pass the function in like this: `onClick={logout}`, then the button should be configured to use that function when it's clicked. Are you passing it in like this: `onClick={logout()}`?